chapter-customizing.asciidoc

```diff
@@ -1356,7 +1357,7 @@ add a plugin element to your build configuration.
 [[customizing-sect-custom-packaged]]
 === Building a Packaged Command Line Application
 
-In the <<customizing-sect-custom-exec>> section earlier in descriptor
+In <<customizing-sect-custom-exec>> earlier in descriptor
```

Comment:
The display name for this section has the word "Section" in it, so the old wording ended up as "in the Section X.X section", which scans really oddly. Sounds redundant.